packages/arb-token-bridge-ui/src/util/ExponentialBackoffUtils.ts

```diff
@@ -1,3 +1,14 @@
+import { backOff as _backOff, BackoffOptions } from 'exponential-backoff'
+
+const backoffOptions: BackoffOptions = {
+  startingDelay: 1_000,
+  timeMultiple: 1.5
+}
+
+export function backOff<T>(request: () => Promise<T>): Promise<T> {
+  return _backOff(request, backoffOptions)
```

Comment:
do we want to turn on full jitter?
https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/

i think we do?